plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -456,13 +439,11 @@ def check_temp_issue(self, port_obj, row, timestamp):
         if not self.temp_check:
             return None
         cable_temp = get_counter(Constants.TEMP_COUNTER, row, default=None)
-        if cable_temp is not None and not pd.isna(cable_temp):
-            if cable_temp in ["NA", "N/A", "", "0C", "0"]:
+        if cable_temp is not None and not numpy.isnan(cable_temp):
+            if cable_temp in ["NA", "N/A", "", "0C"]:
```

Comment:
"0" option was add as part of this PR. If I'm not wrong, it was reproduced on some testing environment.
https://github.com/Mellanox/ufm_sdk_3.0/pull/193

Reply:
@egershonNvidia can you please add the "0" back to the string array?